Jealous-Seals/The-Task-Tracker
# Rm drag and drop

added code using jquery and jqueryUI to enable dragging and dropping of elements, added functions to handle when cards are dropped on the other article columns.

```
git fetch --all
git checkout rm-drag-and-drop
```

to test
1. make sure you have up to date tasks in local storage, if not uncomment tasks in main.js, run application, then comment them out.
1. move around tasks and make sure they are responding correctly.
